contracts/JBPrices.sol

```diff
@@ -109,12 +97,10 @@ contract JBPrices is IJBPrices, Ownable {
   function addFeedFor(
     uint256 _currency,
     uint256 _base,
-    AggregatorV3Interface _feed
+    IJBPriceFeed _feed
   ) external override onlyOwner {
     // There can't already be a feed for the specified currency.
```

Comment:
Since we know have multiple "oracle provider", wouldn't it be cool to be able to change the feed for a currency (ie if we transition from uniV3 to chainlink for instance)?

Reply:
would be. though its another layer of trust that might instead merit a new currency index despite reporting rates for the same currency. meh?

Reply:
lingering thoughts here @drgorillamd ?

Reply:
Then new terminal/migration/etc, while we are "just" changing oracle? 

But indeed, the safer/trustlesser the better, you're right

Reply:
no need for migration, just new `currency` index in JBFundAccessConstraints https://github.com/jbx-protocol/juice-contracts-v2/blob/main/contracts/structs/JBFundAccessConstraints.sol to use a new feed.